src/exception/types.ts

```diff
@@ -76,6 +77,11 @@ export type EntityException = {
   follow_up: FollowUpExceptionRecord[];
 };
 
+export type EntityExceptionSchemaNames = Extract<
+  ClinicalEntitySchemaNames,
+  ClinicalEntitySchemaNames.SPECIMEN | ClinicalEntitySchemaNames.FOLLOW_UP
+>;
+
```

Comment:
reducing the amount of "reinventing the wheel"  - we have already defined variations on schema names a bunch.
just need a subset of `ClinicalEntitySchemaNames`